webapp/app/routes/room/room.component.js

```diff
@@ -58,11 +67,32 @@ export class Room extends PureComponent {
     this.setState({ open: false });
   };
 
+  renderDrawerFooter() {
+    const { activeRoom, classes, leaveRoom } = this.props;
+
+    return ifElse(
+      complement(equals(DEFAULT_ROOM)),
+      always(
```

Comment:
dziala Ci to w porzadku? jest problem z renderowaniem komponentow za pomoca always, czasami wyrenderuje go nawet jak ten warunek sie nie spelni, bezpieczniej uzyc arrow fn